packages/cms-cli/commands/functions/build.js

```diff
@@ -0,0 +1,73 @@
+const ora = require('ora');
+const {
+  addPortalOptions,
+  addConfigOptions,
+  setLogLevel,
+  getPortalId,
+  addUseEnvironmentOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cms-lib');
+const { logger } = require('@hubspot/cms-lib/logger');
+const { buildPackage } = require('@hubspot/cms-lib/api/functions');
+const { validatePortal } = require('../../lib/validation');
+
+const makeSpinner = (functionPath, portalIdentifier) => {
+  return ora(
+    `Building new bundle for '${functionPath}' on portal '${portalIdentifier}'.\n`
+  );
+};
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validatePortal(options)))) {
+    process.exit(1);
+  }
+};
+
+exports.command = 'build <path>';
+exports.describe =
+  'builds a new dependency bundle for the specified function folder';
+
+exports.handler = async options => {
+  loadAndValidateOptions(options);
+
+  const { path: functionPath } = options;
+  const portalId = getPortalId(options);
+  const spinner = makeSpinner(functionPath, portalId);
+
+  trackCommandUsage('functions-build', { functionPath }, portalId);
+
+  logger.debug(`Starting build for function(s) with path: ${functionPath}`);
```

Comment:
Probably not part of this PR specifically, but I feel like any extra insight we can give to the user that is happening here would he helpful.  

e.g.
```
[spinner] Building myFunctionFolder.functions/function1
```

```
[✅] Building myFunctionFolder.functions/function1
[✅] Building myFunctionFolder.functions/function2
[spinner] Building myFunctionFolder.functions/function3
```

Reply:
Yeah that would be helpful, this is not currently polling, the response is just delayed until the build completes. Something we could def add to vNext @bkrainer

Reply:
Agreed. I see this moving towards a fully asynchronous build process, where we poll based on a build ID. Could maybe even stream the `npm install` output so we could show exactly how far along the build process is